glyphs_plist/src/norad_interop.rs

```diff
@@ -22,10 +25,15 @@ impl From<&norad::Contour> for Path {
     }
 }
 
-impl From<&Path> for norad::Contour {
-    fn from(path: &Path) -> Self {
-        let mut points: Vec<norad::ContourPoint> =
-            path.nodes.iter().map(|node| node.into()).collect();
+impl TryFrom<&Path> for norad::Contour {
+    type Error = norad::error::NamingError;
+
+    fn try_from(path: &Path) -> Result<Self, Self::Error> {
+        let mut points: Vec<norad::ContourPoint> = path
+            .nodes
+            .iter()
+            .map(|node| node.try_into())
+            .collect::<Result<_, _>>()?;
```

Comment:
Nit: I don't like that the type signature is split in two places here

```suggestion
        let mut points = path
            .nodes
            .iter()
            .map(|node| node.try_into())
            .collect::<Result<Vec<norad::ContourPoint>, _>>()?;
```